app/src/main/java/com/example/atheneum/fragments/HomeFragment.java

```diff
@@ -23,6 +24,10 @@ public HomeFragment() {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
+
+        // set title
+        ((MainActivity) getActivity()).setActionBarTitle(getContext().getResources().getString(R.string.app_name));
+

```

Comment:
I don't think that in general it is a good idea to cast your activity to a MainActivity. You assume that the type of the return value of getActivity() will be MainActivity but that isn't true in general when using fragments. I think it would be better if you wrapped this code in a check to "instance of MainActivity" since you can't guarantee that any other activity that implements this will have the actionabar.

Reply:
Fixed